x/community/types/params.go

```diff
@@ -0,0 +1,26 @@
+package types
+
+import "time"
+
+var (
+	DefaultUpgradeTimeDisableInflation = time.Time{}
```

Comment:
Should we use a non-zero/default value for the default time and enforce it in `Validate()`? 

In case we're doing `Time.After()` or `Time.Before()` checks when using the value on switchover, empty value might cause issues since it's considered earlier

Reply:
The inflation upgrade logic will handle unset times by upgrading if unset to ensure vanilla kava chain always have new inflation logic enabled. https://github.com/Kava-Labs/kava/pull/1706/files#diff-15d449f4515711b278439f4600052557555689d56ea115a58619aa3061738906R30

Reply:
Agree with @DracoLi here, new chains should default to new logic to match mainnet behavior after the upgrade time